com.unity.render-pipelines.universal/Editor/Camera/URPSerializedCamera.cs

```diff
@@ -0,0 +1,50 @@
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    internal class URPSerializedCamera
```

Comment:
In HDRP we already have it separated. It is called SerializedHDCamera. We should align to have same naming everywhere I think if the intend is to align separation.

Also, default visibility for a class is already internal. For nested class it would have been private but here the visibility does no change at all.

If the aim is to use the separated class for embedding all the serialization burden, it would make sens to have an Update() and Apply() method that redirect on the serializedObject. This way, the serializedObject can be hidden and all serialization specific code managed here. Especially, we deal with two set of data: the ones in the camera and the ones in the additionaldatas. You can sync both almost all the time so it give credit to have it here in its own Update/Apply.

The naming right now is a little confusing:
- m_SerializedCamera (name of the variable in code) is in fact the serialization on the additional data component
- settings handle datas of the legacy camera


Note that we do this in HDRP because:
- we have a lot of fields
- we  have really specific serialization process sometimes that need dedicated code.
- it allows to make a clear delimitation between data / drawer / logic. We made this by constructing a static library drawer HDCameraUI and thus the editor logic remain in the Editor class. But it complexity things so as long as class are not that long, I advice against doing this.

I am not 100% sure URP will profit from this change as it is right now.

Reply:
The idea here is preparing URP for the future as well, this will help us identify the common code, and tooltips, etc.

Thank you for your feedback.